### ISSUE TITLE
Breaking March: <picture> default behaviour

### ISSUE THREAD

**Issue by davidnuescheler** [adobe/helix-pages#657]
it would be great to introduce responsive images in helix pages by default.

while there may be a broad range of configuration options, we can get to a place that is a significant improvement over the current behavior by changing the current img tag from eg:

```
            <img src="/hlx_dd76df9c9b121fec5f1b6bc39481247a1f756139.png">
```

to 

```
<picture>
            <source media="(max-width: 400px)" srcset="/hlx_dd76df9c9b121fec5f1b6bc39481247a1f756139.png?width=750&format=webply&optimize=medium" />
            <img loading="lazy" src="/hlx_dd76df9c9b121fec5f1b6bc39481247a1f756139.png?width=2000&format=webply&optimize=medium" alt=""/>
</picture>
```

with the first img set to `loading="eager"`

**Comment by tripodsan** [comment]
> 🎉 This issue has been resolved in version 4.16.0 🎉
> 
> The release is available on [GitHub release](https://github.com/adobe/helix-pages/releases/tag/v4.16.0)
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀

this is not true!

**Comment by davidnuescheler** [comment]
there are some issues with upstream CDNs that don't respect the vary header, so we will not be able to use the `auto=webp` across the board.

the best solution seems to do the following:

for browsers we use `webp`

```
<picture>
            <source media="(max-width: 400px)" srcset="/hlx_dd76df9c9b121fec5f1b6bc39481247a1f756139.png?width=750&format=webply&optimize=medium" />
            <img loading="lazy" src="/hlx_dd76df9c9b121fec5f1b6bc39481247a1f756139.png?width=2000&format=webply&optimize=medium" alt=""/>
</picture>
```

as originally specified, and we will have to use a client side polyfill for the shrinking browsers that don't support `webp` to go to jpeg.

for the meta image used for link expanders / social previews etc. we will have to go with jpg for the the forseeable future. so we have to remove the `auto=webp` and land with something like this.

```
https://blog.adobe.com/hlx_4a0ffbe8665ff1fdd9f21dd6c317812de3207c01.jpeg?format=pjpg&optimize=medium&width=1200
```
